service/matching/matching_engine.go

```diff
@@ -1987,6 +2119,114 @@ func (e *matchingEngineImpl) pollTask(
 	return pm.PollTask(ctx, pollMetadata)
 }
 
+// isValidName checks if each character is a letter/number in the input string
+func isValidName(input string) bool {
```

Comment:
is valid name for what? deployments? then it should be `isValidDeploymentName`.. this is a package scope

Reply:
I am going to alter this function definition by making it a method for `taskQueuePartitionManagerImpl` and change the name to `isValidDeploymentName`.